core/src/main/java/hudson/model/ParametersAction.java

```diff
@@ -298,12 +306,18 @@ public void onAttached(Run<?, ?> r) {
         } else {
             this.parameterDefinitionNames = Collections.emptyList();
         }
+        if (wasInitializedWithNullParameters) {
+            LOGGER.log(Level.WARNING, "ParametersAction has been initialized with null parameter list for Run {0}. It is a bug in the plugin, which created this action (see JENKINS-39495).", r);
```

Comment:
The problem I see with this is that it spams the admin log and the affected users can't even do anything about it. It doesn't even tell them which of the 100+ plugins they have installed.

Remember the Stapler change for `$class`, or whatever that was, that spammed all the logs? We changed it to FINE later because it was just too annoying. And *that* actually told people which plugin was responsible (mostly).